cabal-install/src/Distribution/Client/CmdListBin.hs

```diff
@@ -3,6 +3,7 @@
 {-# LANGUAGE OverloadedStrings #-}
 {-# LANGUAGE RecordWildCards   #-}
 {-# LANGUAGE TupleSections     #-}
+{-# OPTIONS_GHC -Wno-deferred-out-of-scope-variables #-}
```

Comment:
Hmm, this is a bit unexpected. Is this necessary? Why?

Reply:
oops sorry didn't spot that!! it was HLS 🤦